CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ set(CMAKE_OSX_ARCHITECTURES "x86_64")
 # features can be used, not the minimum required version to run.
 set(CMAKE_OSX_DEPLOYMENT_TARGET "10.9" CACHE string "")
 
+set(CMAKE_USER_MAKE_RULES_OVERRIDE "CMake/FlagsOverride.cmake")
```

Comment:
It's a variable pointing to a script that is run after the compiler checks, it's not run those, but it needs to be set before.